activity_calendar/migrations/0020_auto_20220320_2314.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 2.2.15 on 2022-03-20 22:14
+
+from django.db import migrations, models
+
+SLOT_CREATION_STAFF = 'CREATION_STAFF'
+SLOT_CREATION_NONE = "CREATION_NONE"
+
+def forwards_func(apps, schema_editor):
+    # This migration introduces a new slot mode: none. But this was already the name for staff related activities
+    # So we need to shift that in the databse to the new name
+    Activity = apps.get_model("activity_calendar", "Activity")
+    ActivityMoment = apps.get_model("activity_calendar", "ActivityMoment")
+    db_alias = schema_editor.connection.alias
+    Activity.objects.using(db_alias).\
+        filter(slot_creation=SLOT_CREATION_NONE).\
+        update(slot_creation=SLOT_CREATION_STAFF)
+    ActivityMoment.objects.using(db_alias). \
+        filter(local_slot_creation=SLOT_CREATION_NONE). \
+        update(local_slot_creation=SLOT_CREATION_STAFF)
+
+def backwards_func(apps, schema_editor):
+    Activity = apps.get_model("activity_calendar", "Activity")
+    ActivityMoment = apps.get_model("activity_calendar", "ActivityMoment")
+    db_alias = schema_editor.connection.alias
+    Activity.objects.using(db_alias). \
+        filter(slot_creation=SLOT_CREATION_STAFF). \
+        update(slot_creation=SLOT_CREATION_NONE)
+    ActivityMoment.objects.using(db_alias). \
+        filter(local_slot_creation=SLOT_CREATION_STAFF). \
+        update(local_slot_creation=SLOT_CREATION_NONE)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('activity_calendar', '0017_core_activity_groupings'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='activity',
+            name='slot_creation',
+            field=models.CharField(choices=[('CREATION_STAFF', 'Never/By Administrators'), ('CREATION_AUTO', 'Automatically'), ('CREATION_USER', 'By Users'), ('CREATION_NONE', 'No signup')], default='CREATION_AUTO', max_length=15),
```

Comment:
Should probably update the description of the CREATION_STAFF choice (also below)
```suggestion
            field=models.CharField(choices=[('CREATION_STAFF', 'By Administrators'), ('CREATION_AUTO', 'Automatically'), ('CREATION_USER', 'By Users'), ('CREATION_NONE', 'No signup')], default='CREATION_AUTO', max_length=15),
```